### PR TITLE
feat: added signup page

### PR DESCRIPTION
**Pull request by varnita619**
- added signup page

Screenshot (76)

### CODE REVIEW

File: src/Pages/Auth/HomePage.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const HomePage = () => {
           <input
             className="py-1 w-full border"
             type="text"
-            placeholder="Enter Email - test@gmail.com"
+            placeholder="Username"
```
Comment (anuj-kumary) [inline src/Pages/Auth/HomePage.jsx]: Instead of text you can use email here for email username

**Review by anuj-kumary — APPROVED** [review]
Preview looks good, Code is good overall.
Good to merge.

File: src/Pages/Auth/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            name="last name"
+            autoComplete="true"
+            required
+          />
+
+          <input
+            className="py-1 w-full border"
+            type="text"
+            placeholder="Username"
+            name="psw"
```
Comment (anandsr21harshit) [inline src/Pages/Auth/Signup.jsx]: use good names. psw is not that good

**Review by anandsr21harshit — APPROVED** [review]

**Review by Aman-Jain1996 — APPROVED** [review]
Good to merge , 
One suggestion -> Don't use small names for placeholder or name of input field

File: src/Pages/Auth/Signup.jsx (hunk 2 above)
Comment (varnita619) [reply src/Pages/Auth/Signup.jsx]: done

**Pull request merged**